renderer/src/components/ActivityChartCard.jsx

```
@@ -10,11 +10,22 @@
   const [debugInfo, setDebugInfo] = useState(null);
   const [retryCount, setRetryCount] = useState(0);
 
+  // Cleanup function for blob URLs
+  const cleanupBlobUrl = (url) => {
+    if (url && url.startsWith('blob:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)